index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Loader</title>
+</head>
+<body>
+    <section>
```

Comment:
Wrap it inside ```main```

Reply:
I think yes,
as inset is the shorthand way of positioning

Reply:
The `main` was supposed to replace the `section` tag which was inside `body`. How can you wrap the `body` inside the `main`??? Please check which line comments are added!!!! and please be clear of `head` and `body` tags.

Reply:
sorry, let me edit